tests/framework/installer/ceph_installer.go

```diff
@@ -44,11 +44,14 @@ import (
 const (
 	// test with the latest pacific build
 	pacificTestImage = "quay.io/ceph/ceph:v16"
-	// test with the latest pacific build
+	// test with the latest quincy build
 	quincyTestImage = "quay.io/ceph/ceph:v17"
+	// test with the latest reef build
+	reefTestImage = "quay.io/ceph/ceph:v18"
 	// test with the current development version of Pacific
 	pacificDevelTestImage = "quay.io/ceph/daemon-base:latest-pacific-devel"
 	quincyDevelTestImage  = "quay.io/ceph/daemon-base:latest-quincy-devel"
+	reefDevelTestImage    = "quay.io/ceph/ceph:v18"
```

Comment:
Here is my suggestion... Let's get this PR merged with the following changes:
- Change this image back to `quay.io/ceph/daemon-base:latest-reef-devel`
- Change the default version (line 104 below) back to `QuincyVersion`
- In the daily-nightly-jobs.yaml, remove the `pull_request:` change

This will allow the daily jobs to test against reef and we can see if the expected fix makes it into an image soon. 

Then we can open a separate PR that activates Reef testing by changing the default (on line 104) back to ReefVersion. We will need to wait to merge that PR until after Reef is passing.